persist-cli/src/main/java/io/ballerina/persist/nodegenerator/syntax/constants/BalSyntaxConstants.java

```diff
@@ -48,6 +48,7 @@ private InheritedTypeReferenceConstants() {
     public static final String EQUAL = "=";
     public static final String VALUE = "value";
     public static final String PERSIST_MODULE = "persist";
+    public static final String ENUM = "ENUM";
     public static final String PERSIST_ERROR = "persist:Error";
     //public static final String G_SHEET_CREATE_SQL_RESULTS = "_ = check " +
     //        "self.persistClients.get(%s).runBatchInsertQuery(data);";
```

Comment:
Shall we remove them

Reply:
updated in [cbe1595](https://github.com/ballerina-platform/persist-tools/pull/230/commits/cbe15950867ad5ea4a5fda474cedaf83f86c5b4b)